terragrunt/aws/network/security-groups.tf

```diff
@@ -0,0 +1,61 @@
+# 
```

Comment:
Really minor, but could you name the file `security_groups.tf` with an underscore just so we're consistent with the other `*.tf` file names.

Reply:
Done!